core/src/main/python/create.py

```diff
@@ -341,7 +341,9 @@ def validateRCUArgsAndModel(model_context, model):
         has_atpdbinfo = atp_helper.has_atpdbinfo(rcu_db_info)
 
         if model_context.get_archive_file_name() and not has_regular_db:
-            os.environ['oracle.jdbc.fanEnabled'] = 'false'
+            import java.lang.System as javasys
```

Comment:
PEP-8 has warning about this - can import be at the top?

Reply:
Yes, already made the change